ci-operator/step-registry/openshift-qe/orion/olmv1/openshift-qe-orion-olmv1-commands.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
+set -x
+
+if [ ${RUN_ORION} == false ]; then
+  exit 0
+fi
+
+python --version
+pushd /tmp
+python -m virtualenv ./venv_qe
+source ./venv_qe/bin/activate
+
+send_slack_notification() {
+  local jobID="$1"
+
+  SLACK_WEBHOOK_URL=$(cat /var/run/vault/mirror-registry/olm_slack_channel 2>/dev/null)
+  if [[ -z "${SLACK_WEBHOOK_URL:-}" ]]; then
+    echo "Error: SLACK_WEBHOOK_URL environment variable is required."
+    return 1
+  fi
+
+  export PYTHONPATH="/go/src/github.com/openshift/openshift-tests-private/hack/:${PYTHONPATH:-}"
+
+  python3 - <<EOF
+from slack_notify import SlackClient
+import os
+
+webhook_url = os.getenv("SLACK_WEBHOOK_URL")
+notificationList = [
+    "*OLMv1 Performance Abnormal Notification* :zap:",
+    "Prow Job: ${jobID}"
+]
+
+client = SlackClient()
+client.notify_to_slack(webhook_url, notificationList)
+EOF
+}
+
+notify_slack_if_failure() {
+  local xml_file="$1"
+
+  if [[ ! -f "$xml_file" ]]; then
+    echo "Error: File $xml_file not found" >&2
+    return 1
+  fi
+
+  local failures
+  failures=$(sed -n 's/.*failures="\([0-9]*\)".*/\1/p' "$xml_file" | head -n1)
+
+  echo "[DEBUG] Detected failures = '$failures'"
+
+  if [[ -z "$failures" || ! "$failures" =~ ^[0-9]+$ ]]; then
+    echo "[ERROR] Unable to parse failure count from XML." >&2
+    return 1
+  fi
+
+  if [[ "$failures" == "0" ]]; then
```

Comment:
@jianzhangbjz it should be `if [[ "$failures" != "0" ]]; then`

Reply:
Yes, just for debugging. Will change it back once ready.